### ISSUE TITLE
count function in aggregations should work even without a selection set

### ISSUE THREAD

**Issue by HeyItsJs** [spaceuptech/space-cloud#1063]
For using aggregation functions (`min`, `max`, `sum`, etc.) we need to provide a selection set inside the aggregation functions denoting the fields on which we want to perform the aggregation function.

For example:

```graphql
{
  expenditures @db {
    aggregate {
      sum {
        amount
      }
    }
  }
}
```

For example, the above aggregation query means that we want the sum of the `amount` inside the `expenditures` table. 

If we don't provide the field inside the aggregation function (`amount` in the above example), then we get the `null` as the value for `aggregate`. This makes complete sense as the `sum` function needs a column/field.

However, for the `count` function there should be no such restriction, we should allow a `count` aggregation even if a field is not specified inside it.  Meaning this query should ideally work:

```graphql
{
  expenditures @db {
    aggregate {
      count
    }
  }
}
```

In the SQL world, this is equivalent of:
  
```sql
select count(*) from <table-name>;
```

Basically, we should use `*` if a field is not specified inside the `count` function
